### DISCUSSION TITLE
Patience Parameter in Trainer

### DISCUSSION THREAD

**Discussion by danielkorth** [Lightning-AI/pytorch-lightning#7557] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hey, 
Im wondering why the patience parameter has been removed from the Trainer class.
Im pretty sure that there once was this parameter and I used it a lot. 
Has it just been moved to another class, or is there a workaround such that I can still use patience when training my Model?

**Comment by tshu-w** [comment, marked as answer]
You can use [EarlyStopping Callback](https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.callbacks.early_stopping.html?highlight=early_stopping).
